package.json

```diff
@@ -25,7 +25,9 @@
     "standard-wdio": "wdio run ./tests/e2e/standard/wdio.conf.js",
     "default-wdio-mobile": "wdio run ./tests/e2e/default-mobile/wdio.conf.js",
     "local-images": "export VERSION=$(node ./scripts/build/get-version.js) && grunt build-service-images && grunt create-local-docker-compose-files",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "cd webapp && ng cache clean && cd ..",
+    "dev-install": "npm ci && npm run postinstall"
```

Comment:
I believe `postinstall` will run _every_ time you run `npm ci` automatically. Is this not true?

Reply:
🤦🏽‍♀️😩